realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -346,14 +313,56 @@ protected void onError(LogoutResponse response) {
     }
 
     /**
+     * Change this users password. This is done synchronously and involves the network, so calling this method on the
+     * Android UI thread will always crash.
+     *
+     * @param newPassword the users new password.
```

Comment:
"users" -> "user's"

Is `newPassword` in clear text? Or is encrypted somehow? If it is encrypted, encryption method should be mentioned. If not encrypted, it is insanely insecure!!!!